test/plugins/features/movie/controller.test.js

```diff
@@ -13,12 +13,49 @@ describe('movie controller', () => {
       return Controller.create(payload)
       .then((movie) => {
         expect(movie.get('title')).to.eql(payload.title);
+      });
+    });
 
-        return new Movie({ id: movie.id }).fetch();
-      })
-      .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
-        expect(movie.get('release_year')).to.eql(payload.release_year);
+  });
+
+  describe('get', () => {
+
+    const movie = { title: 'foo', release_year: 2016 };
```

Comment:
Definitely not something you have to implement in this PR, but this kind of "sample data", we like to use [factories](https://github.com/rosiejs/rosie) to essentially have a "default" schema that we can just generate and overwrite whenever we need to. It also makes it easy when the schema changes and you don't want to have to update 30 different tests that use it.